version_bump.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+DIFF="$(git diff --raw HEAD^1 VERSION)"
+
+echo "$(curl -X PUT -H "content-type: application/json" -d '{"name": "'$1'", "password": "'$2'"}' http://registry.npmjs.org/-/user/org.couchdb.user:$1)"
```

Comment:
What are you trying to do here?
and the echo and subshell look useless

Reply:
Yeah, echo was the part of the code I forgot to remove. I'm trying to login into nmpjs to be able to republish package